plugins/out_s3/s3.c

```diff
@@ -1269,27 +1233,22 @@ static int put_all_chunks(struct flb_s3 *ctx, int is_startup)
         if (fs_stream == ctx->stream_upload) {
             continue;
         }
+
         /* skip metadata stream */
         if (fs_stream == ctx->stream_metadata) {
             continue;
         }
 
+        /* on startup, we only send old chunks in this routine */
+        if (is_startup == FLB_TRUE && fs_stream == ctx->stream_active) {
```

Comment:
TODO: need to double check that I didn't mess up the async flags for startup vs normal execution